std/experimental/ndslice/internal.d

```diff
@@ -513,6 +477,8 @@ private bool isValidPartialPermutationImpl(size_t N)(in size_t[] perm, ref int[N
     return true;
 }
 
+enum toSize_t(size_t i) = i;
+enum isSize_t(alias i) = is(typeof(i) == size_t);
```

Comment:
`std/experimental/ndslice/internal.d` is public because it is required to remove existing ndslice modules from Mir but continue add new ndslice modules. So, it should be shared/public for now.